packages/core/src/Modules/ComplaintsPolicy/Components/complaints-policy-content.jsx

```diff
@@ -123,178 +103,78 @@ const getGeneralQueriesText = landing_company_shortcode => {
-                        />,
-                    ]}
-                />
-            );
-            if (landing_company_shortcode === 'maltainvest') {
-                texts.push(
-                    <Localize
-                        key={texts.length}
-                        i18n_default_text='<0/><1/>You may also raise your unresolved dispute to the <2>Office of the Arbiter for Financial Services</2>.'
-                        components={[
-                            <br key={0} />,
-                            <br key={1} />,
-                            <a
-                                key={2}
-                                className='link link--orange'
-                                rel='noopener noreferrer'
-                                target='_blank'
-                                href='https://www.financialarbiter.org.mt/'
-                            />,
-                        ]}
-                    />
-                );
-
-                if (is_uk) {
-                    texts.push(
-                        <Localize
-                            key={texts.length}
-                            i18n_default_text='<0/><1/>If you reside in the UK and you are unhappy with our response you may escalate your complaint to the <2>Financial Ombudsman Service</2>.'
-                            components={[
-                                <br key={0} />,
-                                <br key={1} />,
-                                <a
-                                    key={2}
-                                    className='link link--orange'
-                                    rel='noopener noreferrer'
-                                    target='_blank'
-                                    href='https://www.financial-ombudsman.org.uk/'
-                                />,
-                            ]}
-                        />
-                    );
-                }
-            }
-            break;
-        }
-    }
-
-    return texts;
-};
-
-const getSubmissionOfAComplaintText = landing_company_shortcode => (
-    <React.Fragment>
+    texts.push(
         <Localize
-            i18n_default_text='To file a complaint about our service, send an email to <0>complaints@deriv.com</0> and state your complaint in detail. Please submit any relevant screenshots of your trading or system for our better understanding.'
+            key={0}
+            i18n_default_text='If you are not satisfied with the outcome, you can escalate your complaint to the <0>Financial Commission</0>.'
             components={[
                 <a
                     key={0}
                     className='link link--orange'
                     rel='noopener noreferrer'
                     target='_blank'
-                    href='mailto:complaints@deriv.com'
+                    href='https://financialcommission.org/resolving-a-dispute/how-to-file-a-complaintdispute/'
```

Comment:
Hi @matin-deriv can we put this URL some where in constants file  😬 
Thank you

Reply:
We can refactor the component in a different PR.

I believe the changes are being shown this way because of GH algorithm.